pkg/manager/internal.go

```diff
@@ -91,8 +91,19 @@ type controllerManager struct {
 	// metricsListener is used to serve prometheus metrics
 	metricsListener net.Listener
 
+	// healthProbeListener is used to serve liveness probe
+	healthProbeListener net.Listener
+
+	// Readiness probe endpoint name
+	readinessEndpointName string
+
+	// Liveness probe endpoint name
+	livenessEndpointName string
```

Comment:
these are pretty standard in kubernetes (e.g. `/healthz` and `/readyz`), so we should default them at the very least